fitnesse/resources/bootstrap-plus/templates/skeleton.vm

```diff
@@ -80,10 +80,18 @@
    #parse( $headerTemplate )
   </header>
 
-  <article>
+
+  <article #if(!$bodyClass.contains("edit")) style="min-height: 690px" #end >
    #parse( $mainTemplate )
   </article>
-
+ #if($bodyClass.contains("edit"))
+ <div id="tooltip-div-edit">
+ #else
+ <div id="tooltip-div">
+  #end
+<img src="/files/fitnesse/bootstrap-plus/img/hint.png">
```

Comment:
Kan de hint niet via css in een before pseudo? Blijft de template wat cleaner

Reply:
wist niet dat dit een mogelijkheid was, ga er even naar kijken.